mvflow-core/src/main/kotlin/net/pedroloureiro/mvflow/MVFlow.kt

```diff
@@ -197,26 +195,27 @@ private class MVFlowImpl<State, Action, Mutation, Effect>(
     private val defaultLogger: Logger = {}
 ) : MVFlowWithEffects<State, Action, Effect> {
     private val state = MutableStateFlow(initialState)
-    private val externalEffectChannel = BroadcastChannel<Effect>(Channel.BUFFERED)
+    private val externalEffectFlow =
+        MutableSharedFlow<Effect>(extraBufferCapacity = 64/*Same as CHANNEL_DEFAULT_CAPACITY */)
     private val mutex = Mutex()
 
     private inner class LoggingEffectSender(private val logger: Logger) : EffectSender<Effect> {
         override suspend fun send(effect: Effect) {
             logger.invoke("Sending external effect $effect")
-            externalEffectChannel.send(effect)
+            externalEffectFlow.emit(effect)
         }
 
         override fun offer(effect: Effect): Boolean {
             logger.invoke("Offering external effect $effect")
-            return externalEffectChannel.offer(effect).also { accepted ->
+            return externalEffectFlow.tryEmit(effect).also { accepted ->
                 if (!accepted) {
                     logger.invoke("Channel rejected previous effect!")
                 }
             }
         }
     }
 
-    override fun observeEffects() = externalEffectChannel.openSubscription().consumeAsFlow()
+    override fun observeEffects() = externalEffectFlow
```

Comment:
Probably should use [`asSharedFlow`](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/as-shared-flow.html) ?